### PR TITLE
fix: css as per designs + SSR build

### PR DESCRIPTION
**Pull request by avirati**
1. Removed background from tabs container
2. Use `window` inside of `useEffect` to fix SSR build

### CODE REVIEW

File: src/components/tabs/TabsTriggerList.tsx [hunk 1]
```diff
@@ -73,7 +74,7 @@ export const TriggerListWrapper: React.FC<ListProps> = ({
   const triggerListRef = useRef<HTMLDivElement>(null)
   const [showLeftScroller, setShowLeftScroller] = useState<boolean>(false)
   const [showRightScroller, setShowRightScroller] = useState<boolean>(false)
-  const [screenWidth, setScreenWidth] = useState<number>(window.innerWidth)
+  const [screenWidth, setScreenWidth] = useState<number>(0)
```
Comment (thomasdigby) [inline src/components/tabs/TabsTriggerList.tsx:77]: Do we need to update this on initial render? I can't see where that would happen otherwise

Comment (avirati) [reply src/components/tabs/TabsTriggerList.tsx:77]: Don't think we do, can be left undefined 👍

Comment (avirati) [reply src/components/tabs/TabsTriggerList.tsx]: Updated to 
`const [screenWidth, setScreenWidth] = useState<number>()`

File: src/components/tabs/TabsTriggerList.tsx [hunk 2]
```diff
@@ -73,7 +74,7 @@ export const TriggerListWrapper: React.FC<ListProps> = ({
   const triggerListRef = useRef<HTMLDivElement>(null)
   const [showLeftScroller, setShowLeftScroller] = useState<boolean>(false)
   const [showRightScroller, setShowRightScroller] = useState<boolean>(false)
-  const [screenWidth, setScreenWidth] = useState<number>(window.innerWidth)
+  const [screenWidth, setScreenWidth] = useState<number>()
```
Comment (thomasdigby) [inline src/components/tabs/TabsTriggerList.tsx:77]: Sorry, I wasn't clear enough in my comment, where are we setting `screenWidth` on initial render so we know whether to render the action icons or not? If we load this view on a narrow viewport, will the action icons show correctly, or do you need to resize the browser to trigger it?

Comment (avirati) [reply src/components/tabs/TabsTriggerList.tsx:77]: There is no manual resizing involved. This `useEffect` which listens to change on `screenWidth`, which in turn shows / hides the scroll button action icons

```typescript
useEffect(() => {
    const triggerList = triggerListRef.current
    if (triggerList) {
      const { offsetWidth, scrollWidth } = triggerList

      const shouldShowScroller = scrollWidth > offsetWidth

      // Scroll to left on resize, hide left scroll button and then decide if right scroll button should be visible or not
      triggerList.scroll?.({
        left: 0
      })
      setShowLeftScroller(false)
      setShowRightScroller(shouldShowScroller)
    }
  }, [screenWidth])
```

Comment (thomasdigby) [reply src/components/tabs/TabsTriggerList.tsx:77]: Ah got it thanks, I misinterpreted when this `useEffect` would run. Looks all good to me 👍

**Review by thomasdigby — APPROVED** [review]

**Review by hollg — APPROVED** [review]

**Pull request merged**